packages/models/src/sessionEvents.ts

```diff
@@ -79,7 +82,10 @@ export class SessionEvents {
     if (sessionIds && !isEmpty(sessionIds))
       builder.whereIn(this.column("session_id"), sessionIds);
 
-    const rows = await builder.then();
+    if (events && !isEmpty(events))
+      builder.whereIn(this.column("type"), events);
+
+    const rows = await builder.orderBy("created_at", "asc").then();
```

Comment:
- Don't use the column names direcly. Make use of the `this.column` util. 
- Also, order by the `id` as well incase two rows were created at the same time. [Here is an example](https://github.com/litespace-org/litespace/blob/2f6eb6c8fa73f82aa0d00eea54c08713f9891c4e/packages/models/src/messages.ts#L146-L149).